test/e2e/specs/editor/various/block-bindings.spec.js

```diff
@@ -1,2451 +0,0 @@
-					name: 'core/buttons',
-					innerBlocks: [
-						{
-							name: 'core/button',
-						},
-					],
-				} );
-				await editor.canvas
-					.getByRole( 'document', {
-						name: 'Block: Button',
-						exact: true,
-					} )
-					.getByRole( 'textbox' )
-					.click();
-				await page
-					.getByRole( 'tabpanel', {
-						name: 'Settings',
-					} )
-					.getByLabel( 'Attributes options' )
-					.click();
-				const urlAttribute = page.getByRole( 'menuitemcheckbox', {
-					name: 'Show url',
-				} );
-				await expect( urlAttribute ).toBeVisible();
-				const textAttribute = page.getByRole( 'menuitemcheckbox', {
-					name: 'Show text',
-				} );
-				await expect( textAttribute ).toBeVisible();
-				const linkTargetAttribute = page.getByRole(
-					'menuitemcheckbox',
-					{
-						name: 'Show linkTarget',
-					}
-				);
-				await expect( linkTargetAttribute ).toBeVisible();
-				const relAttribute = page.getByRole( 'menuitemcheckbox', {
-					name: 'Show rel',
-				} );
-				await expect( relAttribute ).toBeVisible();
-			} );
-		} );
-
-		test.describe( 'Image', () => {
-			test.beforeAll( async ( { requestUtils } ) => {
-				const customFieldMedia = await requestUtils.uploadMedia(
-					path.join(
-						'./test/e2e/assets',
-						'1024x768_e2e_test_image_size.jpeg'
-					)
-				);
-				imageCustomFieldSrc = customFieldMedia.source_url;
-			} );
-
-			test.beforeEach( async ( { editor, page, requestUtils } ) => {
-				const postId = await editor.publishPost();
-				await requestUtils.rest( {
-					method: 'POST',
-					path: '/wp/v2/posts/' + postId,
-					data: {
-						meta: {
-							url_custom_field: imageCustomFieldSrc,
-						},
-					},
-				} );
-				await page.reload();
-			} );
-			test( 'should show the value of the custom field when url is bound', async ( {
-				editor,
-			} ) => {
-				await editor.insertBlock( {
-					name: 'core/image',
-					attributes: {
-						anchor: 'image-url-binding',
-						url: imagePlaceholderSrc,
-						alt: 'default alt value',
-						title: 'default title value',
-						metadata: {
-							bindings: {
-								url: {
-									source: 'core/post-meta',
-									args: { key: 'url_custom_field' },
-								},
-							},
-						},
-					},
-				} );
-				const imageBlockImg = editor.canvas
-					.getByRole( 'document', {
-						name: 'Block: Image',
-					} )
-					.locator( 'img' );
-				await expect( imageBlockImg ).toHaveAttribute(
-					'src',
-					imageCustomFieldSrc
-				);
-
-				// Check the frontend uses the value of the custom field.
-				const previewPage = await editor.openPreviewPage();
-				const imageDom = previewPage.locator(
-					'#image-url-binding img'
-				);
-				await expect( imageDom ).toHaveAttribute(
-					'src',
-					imageCustomFieldSrc
-				);
-				await expect( imageDom ).toHaveAttribute(
-					'alt',
-					'default alt value'
-				);
-				await expect( imageDom ).toHaveAttribute(
-					'title',
-					'default title value'
-				);
-			} );
```

Comment:
Unify all the image attributes in this test for custom sources: https://github.com/WordPress/gutenberg/pull/65526/files#diff-70af8def41b67ce618e94f6b4f68044bb4540816a1a8084a1ba1eac9e0c26595R141-R215

It is missing the title.